server/repositories/cmsQueries/seriesHeaderPageQuery.js

```diff
@@ -19,7 +19,7 @@ class SeriesHeaderPageQuery {
   }
 
   getKey() {
-    return getCmsCacheKey('seriesHeaderPage', this.establishmentName);
+    return getCmsCacheKey('seriesHeaderPage', this.location);
```

Comment:
This was the bit causing the issue right?

Reply:
Yeah... there were a few tbh. I screwed up a rebase :-(